dev.neovide.neovide.yml

```diff
@@ -0,0 +1,233 @@
+id: dev.neovide.neovide
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: neovide
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host
+  - --filesystem=/tmp
+  - --filesystem=/var/tmp
```

Comment:
why?

Reply:
same as above, written by @sqwxl

Reply:
Did you verify if this is needed? If not then remove it.

Reply:
removed https://github.com/flathub/flathub/pull/5745/commits/d5b40c8a74f0a438f5a3faccaf1a80e907f19534